backend/lib/endpoints/users.js

```diff
@@ -190,6 +191,26 @@ async function routes(app) {
       return new User(userData).save();
     },
   );
+
+  app.delete("/", { preValidation: [app.authenticate] }, async (req) => {
+    const { userId } = req;
+    // Insufficient scope to delete users on Auth0
+    // expected any of: delete:users,delete:current_user
+    // Please add any of the needed scopes and uncomment the line below
+    // await Auth0.deleteUser(getCookieToken(req), userId);
```

Comment:
This seems like something that @joshmorel  will be able to help with, and I am sure it has to do with Auth0.

Reply:
For sure @idjevm + thanks @Zaydme . I'll take a review tomorrow and look into this other aspect.

Many sites give you like some time period to recover if you delete and regret it... But I think for a now a full delete is okay

Reply:
Yeah, but I think this is more clearer as of now though. We would have to use the same design for the organisation and user profile so updates will be needed if we do move forward with this one.

Reply:
Any updates about the needed auth0 scopes `delete:users,delete:current_user` ?
As documented [here](https://auth0.com/docs/api/management/v2#!/Users/delete_users_by_id), the `
delete /api/v2/users/{id}` endpoint requires the individual user to have **delete:users** scope, but it's a little bit dangerous, so **delete:current_user** should be good.